admin/views/tabs/network/restore-site.php

```diff
@@ -13,6 +13,9 @@
 	exit();
 }
 
+echo '<div class="tab-block yoast-modal__section">';
```

Comment:
Class `yoast-modal__section` is used for the modal only, not for this. `tab-block` should be enough.